amaranth/tracer.py

```diff
@@ -44,13 +45,13 @@ def get_var_name(depth=2, default=_raise_exception):
             return code.co_names[imm]
         elif opc == "STORE_FAST":
             imm |= int(code.co_code[index + 1])
-            if sys.version_info >= (3, 11):
+            if sys.version_info >= (3, 11) and platform.python_implementation() != 'PyPy':
```

Comment:
Since we are using a private method, should we check for CPython (include) rather than PyPy (exclude)?

Reply:
Yeah makes sense.